packages/mobile/src/app/App.tsx

```diff
@@ -100,22 +101,24 @@ const App = () => {
               <PersistGate loading={null} persistor={persistor}>
                 <ThemeProvider>
                   <WalletConnectProvider>
-                    <PortalProvider>
-                      <ErrorBoundary>
-                        <NavigationContainer>
-                          <Toasts />
-                          <Airplay />
-                          <RootScreen />
-                          <Drawers />
-                          <Modals />
-                          <Audio />
-                          <OAuth />
-                          <NotificationReminder />
-                          <RateCtaReminder />
-                          <PortalHost name='ChatReactionsPortal' />
-                        </NavigationContainer>
-                      </ErrorBoundary>
-                    </PortalProvider>
+                    <GestureHandlerRootView style={{ flex: 1 }}>
```

Comment:
why was the style needed here?

Reply:
its kinda odd, they just recommend doing that so the view takes up the full screen. just to avoid a footgun i went with it, even though i think prob not needed cause our views later on should have flex: 1 anyway

Reply:
https://docs.swmansion.com/react-native-gesture-handler/docs/fundamentals/installation#js